diffrax/_custom_types.py

```diff
@@ -53,34 +52,41 @@
 BufferDenseInfos = dict[str, PyTree[eqxi.MaybeBuffer[Shaped[Array, "times ..."]]]]
 sentinel: Any = eqxi.doc_repr(object(), "sentinel")
 
-LevyArea: TypeAlias = Literal["", "space-time"]
 
+class AbstractLevyArea(eqx.Module):
+    dt: eqx.AbstractVar[PyTree]
+    W: eqx.AbstractVar[PyTree]
 
-class LevyVal(eqx.Module):
+
+class TimeLevyArea(AbstractLevyArea):
+    dt: PyTree
+    W: PyTree
```

Comment:
I think I'd probably call this something like `BrownianIncrement`. (And if you object to that subclassing something called `AbstractLevyArea` then we could change the name to something like `AbstractBrownianReturn`.)

There really isn't any Levy area here so I don't think we should use that in the name.

Reply:
How about renaming `AbstractLevyArea` to `AbstractLevyVal` and then having `LevyIncrement`?

Reply:
I think it's really the "Levy" I'm objecting to here, which is usually taken to refer to some kind of quantity that is second-or-higher-order.

Reply:
I see, I was going to say, based on @andyElking's comment https://github.com/patrick-kidger/diffrax/pull/364#issuecomment-1938722145 `SpaceLevyArea` would be my preference, but you're far more informed on these matters then myself so I'm more than happy to do the `BrownianIncrement` if you think that is the best option.

Reply:
> I see, I was going to say, based on @andyElking's comment [#364 (comment)](https://github.com/patrick-kidger/diffrax/pull/364#issuecomment-1938722145) `SpaceLevyArea` would be my preference, but you're far more informed on these matters then myself so I'm more than happy to do the `BrownianIncrement` if you think that is the best option.

I suppose calling this "Levy" might not be the most relevant anyway. It's just different terms in the signature of the path (t, W). The terms space-time and space-time-time Levy areas were introduced by James and the signature by Terry. So frankly we could easily name this after someone else...

But anyway, any control could have a signature attached to it. So maybe instead of doing this specifically for a BM we could just allow a general control to either just return its value (as with `use_levy=False`), or to also be endowed with some terms of its signature. And then we can just have a SignatureVal instead of a LevyVal?

Reply:
Sometimes we also find ourselves considering functions-of-the-signature-of-the-path. E.g. "full" Levy area itself is a difference of two terms.

Indeed, I think one could think about generalising what we're doing here to handle arbitrary terms in the signature. I don't think we need to worry about that here though: we can always just pursue the same strategy as we already have here, namely to have the control return some kind of special type indicating that.

Anyway, to bring this full-circle to this particular comment: I think let's call this thing `BrownianIncrement`.